starter/schemaorg/README.md

```diff
@@ -0,0 +1,140 @@
+Schema.org proposes a standard ontology for various domains. 
+However, different websites for the same domain might use a different set of properties, and provide
+data in different format. 
+Thus, Schema2QA replies on real Schema.org metadata to decide the final schema, and generate training data.
+
+We have prepared source data for the 6 domains by crawling existing Schema.org metadata from 
+Yelp, Hyatt, LinkedIn, IMDb, Last.fm, and Goodreads, respectively. 
+
+
+### Generate Manifest and Value Datasets
+Manifest.tt file contains the manifest of the Q\&A agent to build, including all the queries and properties 
+available for each query. 
+Genie can automatically generate manifest.tt given crawled Schema.org metadata. 
+Use 
+```bash
+make experiment=$(exp) $(exp)/manifest.tt
+```
+
+Manifest.tt also include natural language annotations for queries and their properties, describing
+how they can be referred in natural language. 
+By default, Genie uses the manual annotations declared in `tool/autoqa/schemaorg/manual-anotations.js`. 
+One can disable that by
+```bash
+make experiment=$(exp) mode=base $(exp)/manifest.tt
+```
+in which case, only a simple heuristics-based algorithm is used to generate the annotation based on 
+the name of the property. 
+
+Genie can also automatically generate natural language annotations using pretrained language models. 
+To enable that, run 
+```bash
+make experiment=$(exp) mode=auto $(exp)/manifest.tt
+```
+
+One can also fine-tune natural language annotations, by updating
+`MANUAL_PROPERTY_CANONICAL_OVERRIDE` or `MANUAL_PROPERTY_CANONICAL_OVERRIDE_BY_DOMAIN` in 
+tool/autoqa/schemaorg/manual-annotations.js`. 
+
+### Generate a dataset
+
+Use:
+```
+make experiment=$(exp) datadir
+```
+To generate a dataset.
+
+The starter code is tuned to generate a full dataset, which will consume a lot of memory and take a long time.
+Use the following options to control the dataset size:
+```
+make experiment=... max_depth=8 target_pruning_size=500 datadir
+```
+Set a smaller depth or pruning size for faster generation.
+
+### Train
+
+Use:
+```
+make experiment=$(exp) model=$(model) train
+```
+Set `model` to a unique identifier of the model. By default, the model is called "1".
+
+Training takes about 7 hours on a single V100 GPU.
+The model is saved in `$(exp)/models/$(model)`.
+
+### Evaluate
+
+The starter code includes evaluation data crowd sourced from Amazon MTruk for the existing
+6 domains. `multidomain` experiments contains the evaluation data for all 6 domains. 
+```bash
+make experiment=$(exp) model=$(model) evaluate
+```
+
+If you obtain your own , you can add it to `$(experiment)/eval/annotated.tsv` for the dev set,
+and `$(experiment)/test/annotated.tsv` for the test set.
+
+Finally, you can evaluate a specific with
+```
+make experiment=... eval_set=... evaluate
+```
+`eval_set` should be "eval" or "test".
+
+Data added to the dev set will be also used during training for cross-validation.
```

Comment:
This line seems spurious? Or it should be moved to the paragraph that goes "you can add it ..."